config/crd/bases/keda.sh_scaledobjects.yaml

```diff
@@ -138,6 +138,8 @@ spec:
                                   - For scale down: 300 (i.e. the stabilization window
                                   is 300 seconds long).'
                                 format: int32
+                                maximum: 3600
```

Comment:
We shouldn't hardcode values in here, even through `vendor`. Next deps update will remove your fix.


Could you please create a patch for kustomize instead? You can check existing ScaledJob patch for reference:
https://github.com/kedacore/keda/blob/main/config/crd/patches/scaledjob_patch.yaml
https://github.com/kedacore/keda/blob/b7c3d24cf37812998a2e2773d8823b7fb96b9331/config/crd/kustomization.yaml#L12

Reply:
Thanks @zroubalik . Have addressed them